src/type/definition.js

```diff
@@ -483,6 +483,7 @@ export type GraphQLResolveInfo = {
   rootValue: mixed,
   operation: OperationDefinition,
   variableValues: { [variableName: string]: mixed },
+  executionPath: Array<string | number>
```

Comment:
This would be more accurately called `responsePath` since it describes the path the value occurs in the response, rather than the charted path in the executor. Though I think simply `path` is descriptive enough.